apps/backend/migrations/20230725092535-change-group-names-to-unique.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.sequelize.query(`
+    CREATE OR REPLACE PROCEDURE update_group_names_transaction()
+    LANGUAGE plpgsql
+    AS $$
+    DECLARE
+      check_name boolean := true;
```

Comment:
use consistent naming patterns.  i'm not sure what it is for sql, but a mix of stringcase and camelcase at teh same time probably isn't it.

also use consistent capitalization - why is boolean all lower case whereas all the others are all caps